iot_config/src/org.rs

```diff
@@ -122,14 +124,23 @@ pub async fn create_org(
             .map(|_| ())?
     };
 
-    insert_constraints(oui as u64, net_id, devaddr_range, &mut txn)
-        .await
-        .map_err(|_| {
-            OrgStoreError::SaveConstraints(format!(
-                "{} - {}",
-                devaddr_range.start_addr, devaddr_range.end_addr
-            ))
-        })?;
+    if is_helium_netid(&net_id) {
+        insert_helium_constraints(oui as u64, net_id, devaddr_ranges, &mut txn).await
+    } else {
+        let constraint = devaddr_ranges
+            .first()
+            .ok_or(OrgStoreError::SaveConstraints(
+                "no devaddr constraints supplied".to_string(),
+            ))?;
+        if check_roamer_constraint_count(net_id, &mut txn).await? == 0 {
+            insert_roamer_constraint(oui as u64, net_id, constraint, &mut txn).await
+        } else {
+            return Err(OrgStoreError::SaveConstraints(format!(
+                "constraint already in use {constraint:?}"
+            )));
+        }
+    }
+    .map_err(|_| OrgStoreError::SaveConstraints(format!("{devaddr_ranges:?}")))?;
```

Comment:
you sure no important error details are being masked with this type of remapping  ( here and in other similar instances )?